models/DataObject/AbstractObject.php

```diff
@@ -79,6 +82,13 @@ abstract class AbstractObject extends Model\Element\AbstractElement
      */
     protected static $disableDirtyDetection = false;
 
+    /**
+     * @internal
+     *
+     * @var string[]
+     */
+    protected static $objectColumns = ['o_id', 'o_parentid', 'o_type', 'o_key', 'o_classid', 'o_classname'];
```

Comment:
Any specific reason why `o_path` is missing?

Reply:
You already did getByPath so I excluded it. 
If you think that it should be here then do i delete that from abstract object?

Reply:
I would just add it to the list so that you have both options to get object as we already have `getById` as well.

Reply:
one more thing.. how getBy`o_parent`, `o_type`, `o_classid` & `o_classname` suppose to work? as this yield all objects

Reply:
Same as other autogenerated methods from class. If there is one object u get that object back, if there is more then we return listing.

Reply:
I added o_path